src/main/java/org/mifos/identityaccountmapper/service/AccountLookupService.java

```diff
@@ -105,34 +105,23 @@ public void accountLookup(String callbackURL,String payeeIdentity, String paymen
         if(!paymentModalityCodes.contains(paymentModality)){
             paymentModality = getValueByKey(paymentModality);
         }
-
-        if(accountValidationEnabled) {
-            AccountValidationService accountValidationService = null;
-            try {
-                accountValidationService = (AccountValidationService) this.applicationContext.getBean(accountValidatorConnector);
-            } catch (NoSuchBeanDefinitionException ex) {
-                // Handle the case when the bean is not found in the application context
-            }
-            assert accountValidationService != null;
-            Boolean accountValidate = accountValidationService.validateAccount(paymentModalityDetails.getDestinationAccount(),
-                    paymentModalityDetails.getInstitutionCode(), fetchPaymentModality(paymentModality),payeeIdentity, callbackURL);
-            if(callbackEnabled) {
-                sendAccountLookupCallback(callbackURL, accountValidate, payeeIdentity, requestId, registeringInstitutionId);
-            }
-        }
-        else{
-            sendAccountLookupCallback(callbackURL, true, payeeIdentity, requestId, registeringInstitutionId);
+        AccountValidationService accountValidationService = null;
+        try {
+            accountValidationService = (AccountValidationService) this.applicationContext.getBean(accountValidatorConnector);
+        } catch (NoSuchBeanDefinitionException ex) {
+            // Handle the case when the bean is not found in the application context
         }
+        assert accountValidationService != null;
```

Comment:
Don't use assertions in application code.

Reply:
Handle validations in standard way.

Reply:
Added a if condition for the check